base/applications/calc/convert.c

```diff
@@ -75,6 +75,7 @@ static const conv_t conv_ANGLE[] = {
     DECLARE_CONV_UNIT(ANGLE, DEGREES,    "$",       "$")
     DECLARE_CONV_UNIT(ANGLE, GRADIANS,   "$*0.9",   "$/0.9")
     DECLARE_CONV_UNIT(ANGLE, RADIANS,    "$/P*180", "$*P/180")
+    DECLARE_CONV_UNIT(ANGLE, KONA,    "$/P*180", "$*P/180")
```

Comment:
What's this unit? I cannot find it documented on the internet.
Is it expected that it uses the same conversion formulae as the radians?